<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Proyecto de Visi√≥n Artificial</string>
  </property>
  <widget class="QFrame" name="imageFrameS">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="imageFrameD">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="captureButton">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>30</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop Capture</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="colorButton">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>80</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Color Image</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="loadButton">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>130</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Image</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QFrame" name="processingFrame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>831</width>
     <height>91</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QComboBox" name="operationComboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Transform pixel</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Thresholding</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Equalize</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gaussian Blur</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Median Blur</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Linear Filter</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dilate</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Erode</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Apply several...</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="operationLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Select operation&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="gaussianLabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Gaussian width&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="thresholdSpinBox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>120</number>
    </property>
   </widget>
   <widget class="QLabel" name="gaussianLabel_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Threshold&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="kernelButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>40</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Kernel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="operOrderButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>40</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Operation Order</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pixelTButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Pixel Transformation</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="gaussWidthBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>3</number>
    </property>
    <property name="maximum">
     <number>19</number>
    </property>
    <property name="singleStep">
     <number>2</number>
    </property>
    <property name="value">
     <number>3</number>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>440</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-weight:600;&quot;&gt;Image Processing&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>180</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Image</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QFrame" name="histoFrameS">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>260</width>
     <height>150</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>4</number>
   </property>
  </widget>
  <widget class="QFrame" name="histoFrameD">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>280</y>
     <width>260</width>
     <height>150</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>4</number>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>captureButton</tabstop>
  <tabstop>colorButton</tabstop>
  <tabstop>loadButton</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>operationComboBox</tabstop>
  <tabstop>gaussWidthBox</tabstop>
  <tabstop>thresholdSpinBox</tabstop>
  <tabstop>pixelTButton</tabstop>
  <tabstop>kernelButton</tabstop>
  <tabstop>operOrderButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
